src/index.rs

```diff
@@ -136,7 +137,16 @@ impl Index {
       let done = self.index_block(&mut wtx)?;
 
       if block % 1000 == 0 {
+        let commit_start_time = Instant::now();
         wtx.commit()?;
+        let now = Instant::now();
+        log::info!(
+          "{}ms elapsed since previous commit, committed up to block {} in {}ms",
+          (now - last_commit_time).as_millis(),
```

Comment:
To make the two times not overlap, should this be:

```suggestion
          (commit_start_time - last_commit_time).as_millis(),
```

Reply:
Yeah, if we don't want them to overlap that's right - I was thinking of the elapsed time reported to include the block commit time. Probably cleaner to have non-overlapping ranges.